bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -185,24 +169,6 @@ const isPublicChat = chat => (
   chat.userId === 'public'
 );
 
-const sortChats = (a, b) => {
-  let sort = sortChatsByIcon(a, b);
```

Comment:
I am not spotting in the code whether we lost this sorting functionality or it's redone in different way. Could you pls link to it if you're finding it?

Reply:
https://github.com/bigbluebutton/bigbluebutton/commit/7b4f6e22319007dea3ec953081e4ff5ab6eb453a

Reply:
it was removed [here](https://github.com/bigbluebutton/bigbluebutton/pull/11173/files#diff-e940ce330551efc76c493aa7eecfc5bc489142d524ac2eb43d3c47167bc93380L292-L294)